css/cart.css

```diff
@@ -0,0 +1,154 @@
+@import url("/style.css");
+@import url("/css/wishlist.css");
+
+.cart {
+  display: flex;
+  width: 50%;
+}
+
+.cart .img-container {
+  width: 50%;
+}
+
+.cart .img-container img {
+  width: 100%;
+  min-height: 100%;
+  border-radius: 1rem;
+  padding: 1rem;
+}
+
+.cart-info {
+  width: 50%;
+  padding: 1.2rem;
+  background-color: #ffffff;
+}
+
+.title {
+  font-size: 3rem;
+  font-weight: 600;
+}
+
+.price {
+  padding: 1rem 0;
+  letter-spacing: 0.6rem;
+}
+
+.price #nonstrike {
+  font-size: 2rem;
+  padding-right: 1rem;
+}
+
+.price #strike {
+  text-decoration: line-through;
+  font-size: 1.6rem;
+  color: #9a9a9a;
+}
+
+.discount {
+  color: #9a9a9a;
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.quantity {
+  font-size: 2rem;
+}
+
+#quantityno {
+  border: 0.1rem black solid;
+  border-radius: 0.2rem;
+  width: 20%;
+  padding-left: 1rem;
+}
+
+.btn .remove {
+  width: 8rem;
+  height: 2.5rem;
+}
+
+.btn .move {
```

Comment:
here you are using the same code for different classes.

Reply:
done